### DISCUSSION TITLE
aws integration

### DISCUSSION THREAD

**Discussion by rr-estudos-2021** [Netflix/conductor#2436] [Q&A]
Hello friends. Alright?

Please, I would like to ask some questions related to AWS.

I'm creating a new workflow where I will have a task that will send an SQS message to a microservice. I have some doubts:

- How to configure AWS Credential ID?
- How to configure AWS URL? For example, for testing with localstack.
- How do I configure Conductor Server to load contrib module via docker?
- How do I configure my task to send an SQS message to another microservice? Is the example below correct?

{
     "sink": "sqs:buy-queue",
     "asyncComplete": false
}

How conductor knows the information about the account and region created for the queue? Example: https://sqs.us-east-1.amazonaws.com/00534542333/buy-queue

Please, I hope I have help, I looked through all the documentation and I looked for examples and I couldn't find it.

**Comment by bjpirt** [comment]
In AWS itself, you'll need to set a policy on wherever Conductor is running to enable it to send to SQS. When testing locally this isn't so easy - I've made a PR to enable you to run Conductor against localstack for testing purposes but it hasn't been merged (https://github.com/Netflix/conductor/pull/3479)

In terms of how to configure conductor to pull in the modules from contrib, you'll need to add the modules you want in to the build.gradle file before building Conductor, this will pull them in.

I believe you're correct with how to send the message to a queue.

Hope this helps
